.github/CONTRIBUTING.md

```diff
@@ -269,6 +272,38 @@ britecharts
 | **test/tools** | Miscelaneous tools
 
 
+### Helpers
+**axis**
+Exposes `getTimeSeriesAxis`, the method that allows a conditional formatting depending on the time series values.
+
+**color**
+Contains the objects with the color schemas, the color gradients and their human friendly names.
+
+**constants**
+Constants for the project
```

Comment:
Could add more description like `Reusable fixed constants can be used through-ought the project`, something like that

Reply:
Makes sense, thanks!